src/containers/Welcome/Step1/Step1.tsx

```diff
@@ -185,7 +186,7 @@ const Step1 = (p: Wizard.StepProps) => {
       const elem = countryData.find(a => a.value === country);
       if (elem) {
         elem.states.forEach(s => {
-          output.push({ label: s, value: s });
+          output.push({ label: t(`main:regionJP.${s}`, s), value: s });
```

Comment:
選ばれた国(ここでは日本のみ)の各stateの値をja.jsonにあるmain:regionJPと照らし合わせて、日本語に入れ替えています。

Reply:
Instead of hardcoding JP, can we leave it as region.${s}? In case you want to incorporate it in other languages, it will not be good to have a regionJP in Spanish or English

Reply:
I changed it to regionOpt.

Reply:
Ok